packages/core/admin/admin/src/content-manager/components/SelectWrapper/index.js

```diff
@@ -141,10 +141,11 @@ function SelectWrapper({
 
       setIsLoading(true);
 
-      const params = { _limit: state._limit, ...defaultParams };
+      const params = { limit: state._limit, ...defaultParams };
 
       if (state._contains) {
-        params[containsKey] = state._contains;
+        const [fieldName] = containsKey.split('_')
```

Comment:
Nice try! However this won't work if the name of the field contains more than one underscore.


In the content manager the layout handles all the query params, you can take a look at this [PR](https://github.com/strapi/strapi/pull/11898) which fixes the bug.